autoscheduler/scraper/tests/commands/scrape_courses_tests.py

```diff
@@ -0,0 +1,240 @@
+        course_num = "7500S"
+        title = "SPORTS LAW"
+        credit_hours = 3
+        term = "201931"
+
+        # Act
+        parse_course(self.law_section_json)
+
+        # Assert
+        Course.objects.get(dept=subject, course_num=course_num, title=title,
+                           credit_hours=credit_hours, term=term)
+
+        assert True
+
+    def test_parse_course_fills_instructor_and_meeting(self):
+        """ Tests if parse_course also adds an instructor and meeting to the database"""
+
+        # Arrange
+        instructor_id = "John M. Moore"
+        instructor_email = "jmichael@email.tamu.edu"
+        instructor = Instructor(id=instructor_id, email_address=instructor_email)
+
+        meeting_id = 4972230
+        crn = 12323
+        building = "ZACH"
+        begin_time = datetime.time(13, 50, 0)
+        end_time = datetime.time(14, 40, 0)
+        meeting_type = "LEC"
+        meeting_days = [True, False, True, False, True, False, False]
+        section = Section(id=497223, subject="CSCE", course_num=121, section_num=501,
+                          term_code=0, min_credits=0, current_enrollment=0,
+                          max_enrollment=0, instructor=instructor)
+
+        #Act
+        parse_course(self.csce_section_json)
+
+        # Assert
+        Instructor.objects.get(id=instructor_id, email_address=instructor_email)
+        Meeting.objects.get(id=meeting_id, crn=crn, building=building,
+                            meeting_days=meeting_days, start_time=begin_time,
+                            end_time=end_time, meeting_type=meeting_type, section=section)
+        assert True
+
+    def test_parse_section_handles_alphanumeric_section_num(self):
+        """ Tests if parse_section accepts an alphanumeric section_num """
+
+        # Arrange
+        subject = "ENGL"
+        course_num = "210"
+        section_num = "M99"
+        term_code = 202011
+        min_credits = 3
+        max_enroll = 25
+        curr_enroll = 3
+        section_id = 511984
+
+        # Section model requires an Instructor
+        fake_instructor = Instructor(id="Fake", email_address="a@b.c")
+        fake_instructor.save()
+
+        # Act
+        parse_section(self.engl_section_json, fake_instructor)
+
+        # Assert
+        Section.objects.get(id=section_id, subject=subject, course_num=course_num,
+                            section_num=section_num, term_code=term_code,
+                            current_enrollment=curr_enroll, min_credits=min_credits,
+                            max_enrollment=max_enroll, instructor=fake_instructor)
+
+        assert True
+
+    def test_scrape_courses_can_generate_meeting_time(self):
+        """ Tests that scrape_courses.convert_meeting_time can handle a normal time """
+
+        # Arrange/Act
+        time = convert_meeting_time("1230")
+
+        # Assert
+        self.assertEqual(time, datetime.time(12, 30))
+
+    def test_scrape_courses_handles_null_meeting_time(self):

```

Comment:
```suggestion
    def test_convert_meeting_time_handles_null_time(self):
```

For tests like this, where we have like a specific situation we're testing, it would be best if it included what situation we're testing (null input in this case), and what we're expecting (for it to return null). So it would be:  `def test_convert_meeting_time_null_time_does_return_null`, but that is pretty wordy.